build.sbt

```diff
@@ -64,6 +65,7 @@ lazy val core = (project in file("core"))
       "org.apache.spark" %% "spark-core" % sparkVersion % "test" classifier "tests",
       "org.apache.spark" %% "spark-sql" % sparkVersion % "test" classifier "tests",
       "org.apache.spark" %% "spark-hive" % sparkVersion % "test" classifier "tests",
+      "org.apache.hadoop" % "hadoop-aws" % hadoopVersion % "test",
```

Comment:
@zsxwing - how do you feel about including `hadoop-aws` in the default delta-storage package?

now GCS, azure users will have to bring in this dependency.